custom-elements/ElementInternals-accessibility.html

```diff
@@ -58,7 +58,6 @@
   "ariaPosInSet",
   "ariaPressed",
   "ariaReadOnly",
-  "ariaRelevant",
```

Comment:
I'd prefer to comment this one with a pointer to the issue: https://github.com/w3c/aria/issue/1267

Reply:
```diff
-  "ariaRelevant",
+  // "ariaRelevant", // blocked by https://github.com/w3c/aria/issue/1267
```